### DISCUSSION TITLE
distributed system?

### DISCUSSION THREAD

**Discussion by csuXinda** [risingwavelabs/risingwave#2848] [Q&A] [answered]
this project is brilliant,but it seems that I can install it in only one host instead of the distrubuted system according to the guide of installation

**Comment by skyzh** [comment, marked as answer]
We will provide distributed set up in the future. For now, you can use `risedev` to compose multi compute node cluster by yourself. See developer guide for more information.

**Reply by csuXinda** [reply]
thanks for answer!you mean  risingware can processing  one task by using several node in the cluster like the flink  in the future right?

**Reply by lmatz** [reply]
RisingWave already can `process one task by using several nodes in the cluster like Apache Flink` now.

But currently, users need to manually configure to set up a cluster of several nodes by themselves (not that difficult, but not as easy as running a single command). We will provide such convenience in the near future.

**Reply by csuXinda** [reply]
Thanks. Looking forward to the  next version~
